src/Components/Nav/NavDropDown.js

```diff
@@ -14,7 +14,7 @@ const NavDropDown = ({ options }) => {
                     return (
                         <DropdownItem
                             tag={Link}
-                            to={subItem.href}
+                            to={subItem.href + window.location.search}
```

Comment:
We're doing this `+ window.location.search` in 3 different parts (and is only going to grow) do you think we can find a way to make this in one place? some sort of automagically add the lang to the links?

Reply:
automatically no idea, maybe create a method in the lang lib called `decorateUrl` that adds/return the search params or something...